app/models/user.rb

```diff
@@ -419,6 +419,7 @@ def seed_aspects
     self.aspects.create(:name => I18n.t('aspects.seed.friends'))
     self.aspects.create(:name => I18n.t('aspects.seed.work'))
     aq = self.aspects.create(:name => I18n.t('aspects.seed.acquaintances'))
+    aq.chat_enabled = true # enable chat aspect in acquaintances
```

Comment:
mh not sure anymore. I think it was for new users only. If they log-in they have no contacts. with the default on acquaintances you would see the hq account and podmin accounts.

I can remove it, but in my opinion its more user-friendly?!